### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by LyonsDo** [dgerosa/writeapaper#2]
The line (/opt/texlive/texdir/texmf-dist/tex/latex/upquote/upquote.sty
throws an error.  There might be some info in the log but I'm not sure what directory that is in.

**Comment by LyonsDo** [comment]
Nice packsge!!!

**Comment by dgerosa** [comment]
Is this referring to your local latex installation or the github remote compilation? The latter works fine (I just created a test repository from this template).

**Comment by LyonsDo** [comment]
Thanks for the reply Davide.

I  created a test repository without any problem and altered the yml file.
Then I tried running lualatex locally. I thought I had read/write
permissions set in TexWorks but I got that error.
It seemed odd to me that it would get as far as that line without reporting
an earlier error.

I went back and ran lualatex from the command line with shell-escape and it
was fine. So I had a working system
and only revisited the issue several days later.

It turns out that TexWorks needs shell-escape enabled in an unexpected way
$synctexoption
-tex-option=--shell-escape

So no issue with your rather elegant system.

Thanks again

Donal.
